tests/install_AMP.sh

```diff
@@ -4,6 +4,6 @@ ROOT=$PWD
 git clone https://github.com/NVIDIA/apex
 cd apex
 pip install -v --no-cache-dir --global-option="--cpp_ext" --global-option="--cuda_ext" ./
-pip install -v --no-cache-dir ./
+# pip install -v --no-cache-dir ./
```

Comment:
To my understanding, the point of the line within this comment is to install Apex w/o extensions if the build on previous line fails, right?

If we want to leave this as a comment, can we add a note that says "If build with extensions fails, you can run this line to build without extensions"? If we don't make it clear, I don't see a point in leaving this commented out.